src/entry/logEntry.service.ts

```diff
@@ -31,19 +31,40 @@ export class LogEntryService {
         return LogEntryRequestError.wrongBodyData;
       }
 
+      // order
       if (
         typeof sortingOrderDESC !== "boolean" &&
         typeof sortingOrderDESC !== "undefined"
       ) {
         return LogEntryRequestError.wrongBodyData;
       }
 
+      let ip;
+      let text;
+      let regex;
+      let classification;
+      let date;
+
+      // filter
+      if (logEntryData.filters !== undefined) {
+        ip = logEntryData.filters.ip;
+        text = logEntryData.filters.text;
+        regex = logEntryData.filters.regex;
+        classification = logEntryData.filters.classification;
+        date = logEntryData.filters.date;
+      }
```

Comment:
create type Filters with those values as instance variables. No need to assign them to individual varaibles